client/main.js

```diff
@@ -29,73 +29,94 @@ if ( global.wcConnectData ) {
 	setBaseURL( global.wcConnectData.baseURL );
 }
 
-const getRouteClass = ( classNames ) => {
+const getRouteClassName = ( classNames ) => {
```

Comment:
This is a weird function, wondering about the reason for refactoring

Reply:
I needed the class name to be able to use it as a key in the object that stores different Redux stores. This way I can reuse the store for the object of the same type. Witch I think that we should actually not do either way and just use one store - I will bring this for discussion. But that is a bigger change for later.